bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4793,7 +4793,7 @@
         </sql>
     </changeSet>
 
-    <changeSet id="adding-create-teleconsultation-privilege-to-teleconsultation-createLink-role-202212012000" author="BAHMNI">
+    <changeSet id="adding-create-teleconsultation-privilege-to-manage-appointments-role-202212012000" author="BAHMNI">
```

Comment:
@kushboojain, ID should not be this long and this causing the issue.
I just checked the liquibasechangelog table and the maximum length of ID should be 63 chars.

`+---------------+
| Field         | Type         | 
+---------------+
| ID            | varchar(63)  |`